ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -145,41 +145,42 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
                     return;
                 }
 
-                try {
-                    braintreeClient.assertCanPerformBrowserSwitch(activity, THREE_D_SECURE);
-                } catch (BrowserSwitchException exception) {
-                    braintreeClient.sendAnalyticsEvent("three-d-secure.invalid-manifest");
-                    callback.onResult(null, new BraintreeException("AndroidManifest.xml is incorrectly configured or another app " +
-                            "defines the same browser switch url as this app. See " +
-                            "https://developer.paypal.com/braintree/docs/guides/client-sdk/setup/android/v4#browser-switch-setup " +
-                            "for the correct configuration: " + exception.getMessage()));
-                    return;
-                }
-
-                if (configuration.getCardinalAuthenticationJwt() == null && ThreeDSecureRequest.VERSION_2.equals(request.getVersionRequested())) {
-                    callback.onResult(null, new BraintreeException("Merchant is not configured for 3DS 2.0. " +
-                            "Please contact Braintree Support for assistance."));
-                    return;
-                }
-                braintreeClient.sendAnalyticsEvent("three-d-secure.initialized");
-
                 if (ThreeDSecureRequest.VERSION_1.equals(request.getVersionRequested())) {
+                    try {
```

Comment:
Should we even bother maintaining this logic? I know we have this ticket (DTBTSDK-2495) to just error out if the merchant opts into 3DS v1.

Reply:
Yeah that's a good call. I went ahead and updated this PR to throw when 3DS v1 is used.